source/MapShipyardPanel.cpp

```diff
@@ -203,6 +212,11 @@ void MapShipyardPanel::DrawItems()
 				for(const StellarObject &object : selectedSystem->Objects())
 					if(object.HasSprite() && object.HasValidPlanet() && object.GetPlanet()->Shipyard().Has(ship))
 					{
+						double outfitValue = 0.;
+						for(const auto &it : ship->Outfits())
+							outfitValue += object.GetPlanet()->GetLocalRelativePrice(it.first, player.Conditions()) * it.first->Cost() * it.second;
```

Comment:
Now that is a great idea, however pricings of different sellType may not be added to each others, so 3 of them will be required, I'll try to make it an internal behaviour too, meaning if the conditions did not change since last call, we just reuse the same pricings.

Reply:
Actually on that I think keeping the visible pricing in cache would be best, considering its the one that will be used the most, we don't expect half the outfits in a system to be import.

Reply:
Is it alright the way I made it, with a cache for the visible customSale?
I am limited by the fact customSales of different saleType are not compatible in this, but maybe I should just make 3 caches?

Reply:
Done as well.